### PR TITLE
use composer command instead of editing json file

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

**Comment by javiereguiluz** [comment]
Nice! :+1: I'd tweak it a bit and I'd remove the version number to leave it as follows:

```
$ composer require ircmaxell/password-compat
```

**Comment by xabbuh** [comment]
I agree. There's little risk a compatibility package releases a new major version containing BC breaks. Though if we wanted to keep a version, I would use `~1.0` or something similar instead.

**Comment by OskarStark** [comment]
if the version is not necessary i am :+1:

but i remembered this: https://github.com/sonata-project/SonataAdminBundle/pull/2649#discussion-diff-22264493

**Comment by javiereguiluz** [comment]
@OskarStark thanks for the info. I agree with @xabbuh's comments.

**Comment by OskarStark** [comment]
so everything is fine or shall i change something?

**Comment by javiereguiluz** [comment]
I'd use:

```
composer require ircmaxell/password-compat:~1.0
```

**Comment by OskarStark** [comment]
fine for me, will change that

**Comment by OskarStark** [comment]
Done, branch ``2.3`` is ok?

File: cookbook/security/_ircmaxwell_password-compat.rst.inc [hunk 1]
```diff
@@ -3,11 +3,6 @@
...
-    .. code-block:: json
+    .. code-block:: bash
 
-        {
-            "require": {
-                ...
-                "ircmaxell/password-compat": "~1.0.3"
-            }
-        }
+        composer require ircmaxell/password-compat:~1.0
```
Comment (xabbuh) [inline cookbook/security/_ircmaxwell_password-compat.rst.inc]: If I am not mistaken we always prefix shell commands with `$ `.

Comment (OskarStark) [inline cookbook/security/_ircmaxwell_password-compat.rst.inc]: you are right, i fixed it

**Comment by WouterJ** [comment]
I actually do not like the version (note that the comment of me you linked was talking about a root package install, otherwise we often omit the version number). 

However, if we want to use the version, we should change it to this to have Windows compability:
```bash
$ composer require ircmaxell/password-compat "~1.0"
```

**Comment by WouterJ** [comment]
@OskarStark can you please take a look at my comment and update this PR or comment?

**Comment by OskarStark** [comment]
Will change that

**Comment by OskarStark** [comment]
done!

fine?

**Comment by WouterJ** [comment]
Yes, perfect. Thanks!

**Comment by OskarStark** [comment]
Thank you

Am Samstag, 19. Dezember 2015 schrieb Wouter J :

**Comment by JHGitty** [comment]
@OskarStark @WouterJ It has been merged without the word ``composer``, right?
https://github.com/symfony/symfony-docs/commit/079330e8bd58da232809ed9dcc1f785ec8573a87

**Comment by WouterJ** [comment]
Hmm, yes indeed. That has slipped through, can you please submit a PR to fix it?

**Comment by OskarStark** [comment]
done @WouterJ and good catch @JHGitty, thank you :+1:

**Pull request merged**